compiler/circle-mpqsolver/src/pattern/PatternResolver.cpp

```diff
@@ -207,3 +306,44 @@ Q8LayerNormWithQ16VarianceResolver::resolve(const luci::Module *module)
 
   return nodes_params;
 }
+
+std::map<luci::CircleNode *, LayerParam>
+Q8SoftmaxWithQ16SubExpResolver::resolve(const luci::Module *module)
+{
+  if (!module)
+  {
+    throw std::runtime_error("no module for pattern resolving");
```

Comment:
There exist mixed case of first letter. It would be better to use capital for the first one. Please check in other codes and fire another PR for fixes.
```suggestion
    throw std::runtime_error("No module for pattern resolving");
```

Reply:
ok